package.json

```diff
@@ -129,8 +124,7 @@
     "eslint": "^5.12.0",
     "eslint-config-terra": "^2.2.0",
     "html-webpack-plugin": "^3.1.0",
-    "jest": "^23.1.0",
-    "nightwatch": "^0.9.12",
+    "jest": "^23.1.0"
```

Comment:
The famous missing comma of destruction.

Reply:
![](https://media.giphy.com/media/KupdfnqWwV7J6/giphy.gif)